driver/src/main/com/mongodb/util/JSONSerializers.java

```diff
@@ -104,6 +105,7 @@ static ClassMapBasedObjectSerializer addCommonSerializers() {
         serializer.addObjectSerializer(ObjectId.class, new ObjectIdSerializer(serializer));
         serializer.addObjectSerializer(Pattern.class, new PatternSerializer(serializer));
         serializer.addObjectSerializer(String.class, new StringSerializer());
+        serializer.addObjectSerializer(Symbol.class, new SymbolSerializer(serializer));
```

Comment:
Seems that 2.x JSON class doesn't support serializing symbols either, so I see this an an enhancement request unrelated to the description of JAVA-1909.  Let's leave it out of this and report it as a separate JIRA.